x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/alerting/alert_deletion/schedule.ts

```diff
@@ -1127,16 +1360,16 @@ export default function alertDeletionTests({ getService }: FtrProviderContext) {
 
         // schedule the task
         await supertest
-          .post(`${getUrlPrefix(Space1.id)}/api/alerts_fixture/schedule_alert_deletion`)
+          .post(
+            `${getUrlPrefix(Space1.id)}/internal/alerting/rules/settings/_alert_delete_schedule`
+          )
           .set('kbn-xsrf', 'foo')
           .send({
-            isActiveAlertDeleteEnabled: false,
-            isInactiveAlertDeleteEnabled: true,
-            activeAlertDeleteThreshold: 90,
-            inactiveAlertDeleteThreshold: 90,
-            spaceIds: [Space1.id, 'default'],
+            active_alert_delete_threshold: undefined,
+            inactive_alert_delete_threshold: 90,
+            space_ids: [Space1.id, 'default'],
```

Comment:
Could you add a test in which the route throws a 403 if the user has access to one space but not to another?